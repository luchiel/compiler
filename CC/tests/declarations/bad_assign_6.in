void main()
{
int a;
double b;
b %= a;
}