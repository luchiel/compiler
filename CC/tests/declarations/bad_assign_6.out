Error: ParseError:5:7 Both operands must be of type int
